crates/game/src/dino.rs

```diff
@@ -3,13 +3,14 @@ use bevy::{
     asset::AssetServer,
     ecs::schedule::IntoScheduleConfigs,
     input::ButtonInput,
-    log::info,
-    prelude::{Commands, KeyCode, MouseButton, Query, Res, Touches, Transform, With},
+    prelude::{
+        Assets, Commands, KeyCode, MouseButton, Query, Res, ResMut, Touches, Transform, With,
+    },
     sprite::Sprite,
     state::{condition::in_state, state::OnEnter},
     time::{Time, Virtual},
 };
-use bevy_kira_audio::prelude::*;
```

Comment:
to not merge use prelude::* with specific import, we should import what we needed